### DISCUSSION TITLE
1 VM Destiny receving packets from 7 VM Transmiting using DPERF

### DISCUSSION THREAD

**Discussion by PedrovlBarbosa** [baidu/dperf#476] [General]
I hope this message finds you well.

I am writing to inquire about the possibility of setting up a testbed where one destination VM receives packets from seven transmitting VMs. The details of this setup are explained in the attached document. Specifically, I would like to know if DPERF can be utilized to perform these functions.

Thank you for your assistance.

![Screenshot from 2024-07-24 10-14-09](https://github.com/user-attachments/assets/ee8a85d4-9bc7-45a1-8db8-f6bc45533354)

希望您一切安好。

我写信是想询问是否可以设置一个测试平台，其中一个目标虚拟机接收来自七个发送虚拟机的数据包。此设置的详细信息在附件中说明。具体来说，我想知道是否可以使用DPERF来执行这些功能。

感谢您的帮助。

**Comment by pengjianzhang** [comment]
yes.

you just need to put 7 'client' in dperf server's configure file.
